types/arangodb/index.d.ts

```diff
@@ -907,7 +907,7 @@ declare namespace ArangoDB {
         write?: string | string[];
         allowImplicit?: boolean;
     }
-    interface Transaction {
+    interface Transaction<ReturnType = any> {
         collections: TransactionCollections | string[];
         action: (params: object) => void | string;
```

Comment:
This should probably be `(params: object) => ReturnType` now?

Reply:
Yup, that makes sense.